web/annotation_editor_params.js

```diff
@@ -15,6 +15,20 @@
 
 import { AnnotationEditorParamsType } from "pdfjs-lib";
 
+/** @typedef {import("./event_utils.js").EventBus} EventBus */
```

Comment:
Here, and everywhere else throughout the patch, imported typedefs should be placed *before* the regular `import`s.